core/src/main/java/nu/yona/server/messaging/service/FirebaseServiceException.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License, v.
+ * 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.messaging.service;
+
+import java.io.Serializable;
+
+import nu.yona.server.exceptions.YonaException;
+
+public class FirebaseServiceException extends YonaException
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this class to remove "Exception" or correct its inheritance. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS2166)

Reply:
I think you are reporting a false positive

Reply:
Indeed, not the first one of this kind.